docs/reference/commandline/version.md

```diff
@@ -18,38 +18,97 @@ Options:
 
 ## Description
 
-By default, this will render all version information in an easy to read
-layout. If a format is specified, the given template will be executed instead.
+The `version` command prints detailed information about the version of the
+`docker` CLI, the Docker Engine (`dockerd`), as well as other components.
 
-Go's [text/template](https://golang.org/pkg/text/template/) package
-describes all the details of the format.
+By default, this renders all version information in an easy to read
```

Comment:
"easy to read layout" is not neutral in the sense of some users might disagree, I'd steer away from using such expressions. I'd just keep the second sentence like "Use the [`--format`](#format) option to customize the default output layout with a Go template".